Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
It should be fine on the user side of things.

We essentially try really hard to ignore any possible way you can break the `npm` outside the keg, but if you can execute things like `npm install -g mocha` and `npm install -g ios-deploy` that should be reasonable sanity-checking in terms of backwards compatibility with the most recent `node` prior to this release on the user side of things.

The more interesting thing would be trying it with `node@4` or `node@0.12` even with this version of `npm`. That could go pop.